crypto_data.py

```diff
@@ -0,0 +1,3760 @@
+crypto_data = {
```

Comment:
Ah, if I'd known that you were asking for the list for RoboKaywee then I could've told you that the Coinbase API only accepts a few different coin types - only these pairs are supported https://api.pro.coinbase.com/products

Reply:
I can shorten the dict to the 74 codes allowed then, easy.

Reply:
I still think we don't need the dict. We can just try the coinbase request and see if it works - if not then fail gracefully with a chat mesage. That way it's futureproof - we don't have to keep a dict up to date - and also simpler.